addons/rose/addon/components/rose/dropdown/key-value/index.hbs

```diff
@@ -0,0 +1,9 @@
+<dl class='rose-dropdown-key-value rose-dropdown-item'>
```

Comment:
Reused `rose-dropdown-item` class defined [here](https://github.com/hashicorp/boundary-ui/blob/main/addons/rose/addon/styles/rose/components/dropdown/_item.scss#L7) to avoid copying styles to mimic dropdown item.